src/Core/Authorization/GraphQLAuthorizationHandler.cs

```diff
@@ -4,6 +4,7 @@
 using System.Diagnostics.CodeAnalysis;
 using System.Security.Claims;
 using HotChocolate.AspNetCore.Authorization;
+using HotChocolate.Authorization;
```

Comment:
With Hot Chocolate we have abstracted the authorization hooks to allow users to more easily bring in new. providers without the need to rely on ASP.NET core APIs